### PR TITLE
docs: Lint all *.md files and replace product-edc with tractusx-edc

### PR DESCRIPTION
**Pull request by florianrusch-zf**
To provide some documentation for the https://github.com/eclipse-tractusx/eclipse-tractusx.github.io repository we need to lint our markdown files.

This was done by using the npm package: https://www.npmjs.com/package/markdownlint-cli2

```shell
$ npm install -g markdownlint-cli2
$ markdownlint-cli2-config .markdownlint.yaml "**/*.md"
```

The configuration file `.markdownlint.yaml` was configured like the lintting is configured in the eclipse-tractusx.github.io repository. The configuration file itself will be added to the repository with the PR #132 

```yaml
"default": true
# Do not restrict line length: https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#MD013
"MD013": false
# Allow same content on headlines on siblings: https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#MD024
"MD024":
  "siblings_only": true
```

closes #111

### CODE REVIEW

**Comment by florianrusch-zf** [comment]
There are currently still some errors in the files. @stephanbcbauer how should we handle the following linting errors?

```plain
docs/samples/Transfer Data.md:9:65 MD051/link-fragments Link fragments should be valid [Context: "[chapter 1](#1--optional--local-setup)"]
docs/samples/Transfer Data.md:13:4 MD051/link-fragments Link fragments should be valid [Context: "[(optional) Local Setup](#1--optional--local-setup)"]
README.md:1:1 MD033/no-inline-html Inline HTML [Element: a]
README.md:1 MD041/first-line-heading/first-line-h1 First line in a file should be a top-level heading [Context: "<a name="readme-top"></a>"]
README.md:10:1 MD033/no-inline-html Inline HTML [Element: br]
README.md:11:1 MD033/no-inline-html Inline HTML [Element: div]
README.md:12:3 MD033/no-inline-html Inline HTML [Element: a]
README.md:13:5 MD033/no-inline-html Inline HTML [Element: img]
README.md:16:3 MD033/no-inline-html Inline HTML [Element: h3]
README.md:17:3 MD033/no-inline-html Inline HTML [Element: h4]
README.md:19:3 MD033/no-inline-html Inline HTML [Element: p]
README.md:21:5 MD033/no-inline-html Inline HTML [Element: br]
README.md:22:5 MD033/no-inline-html Inline HTML [Element: a]
README.md:22:79 MD033/no-inline-html Inline HTML [Element: strong]
README.md:23:5 MD033/no-inline-html Inline HTML [Element: br]
README.md:24:5 MD033/no-inline-html Inline HTML [Element: br]
README.md:25:5 MD033/no-inline-html Inline HTML [Element: a]
README.md:27:5 MD033/no-inline-html Inline HTML [Element: a]
README.md:29:5 MD033/no-inline-html Inline HTML [Element: a]
README.md:34:1 MD033/no-inline-html Inline HTML [Element: details]
README.md:35:3 MD033/no-inline-html Inline HTML [Element: summary]
README.md:36:3 MD033/no-inline-html Inline HTML [Element: ol]
README.md:37:5 MD033/no-inline-html Inline HTML [Element: li]
README.md:38:7 MD033/no-inline-html Inline HTML [Element: a]
README.md:40:5 MD033/no-inline-html Inline HTML [Element: li]
README.md:41:7 MD033/no-inline-html Inline HTML [Element: a]
README.md:43:5 MD033/no-inline-html Inline HTML [Element: li]
README.md:44:7 MD033/no-inline-html Inline HTML [Element: a]
README.md:45:7 MD033/no-inline-html Inline HTML [Element: ul]
README.md:46:9 MD033/no-inline-html Inline HTML [Element: li]
README.md:46:13 MD033/no-inline-html Inline HTML [Element: a]
README.md:49:5 MD033/no-inline-html Inline HTML [Element: li]
README.md:49:9 MD033/no-inline-html Inline HTML [Element: a]
README.md:57:1 MD033/no-inline-html Inline HTML [Element: p]
README.md:57:19 MD033/no-inline-html Inline HTML [Element: a]
README.md:83:1 MD033/no-inline-html Inline HTML [Element: p]
README.md:83:19 MD033/no-inline-html Inline HTML [Element: a]
README.md:87:1 MD033/no-inline-html Inline HTML [Element: p]
README.md:87:19 MD033/no-inline-html Inline HTML [Element: a]
README.md:97:1 MD033/no-inline-html Inline HTML [Element: p]
README.md:97:19 MD033/no-inline-html Inline HTML [Element: a]
README.md:103:1 MD033/no-inline-html Inline HTML [Element: p]
README.md:103:19 MD033/no-inline-html Inline HTML [Element: a]
```

**Comment by stephanbcbauer** [comment]
> Right now there are still some errors in the files. @stephanbcbauer how should we handle the following linting errors?
> 
> ```
> docs/samples/Transfer Data.md:9:65 MD051/link-fragments Link fragments should be valid [Context: "[chapter 1](#1--optional--local-setup)"]
> docs/samples/Transfer Data.md:13:4 MD051/link-fragments Link fragments should be valid [Context: "[(optional) Local Setup](#1--optional--local-setup)"]
> README.md:1:1 MD033/no-inline-html Inline HTML [Element: a]
> README.md:1 MD041/first-line-heading/first-line-h1 First line in a file should be a top-level heading [Context: "<a name="readme-top"></a>"]
> README.md:10:1 MD033/no-inline-html Inline HTML [Element: br]
> README.md:11:1 MD033/no-inline-html Inline HTML [Element: div]
> README.md:12:3 MD033/no-inline-html Inline HTML [Element: a]
> README.md:13:5 MD033/no-inline-html Inline HTML [Element: img]
> README.md:16:3 MD033/no-inline-html Inline HTML [Element: h3]
> README.md:17:3 MD033/no-inline-html Inline HTML [Element: h4]
> README.md:19:3 MD033/no-inline-html Inline HTML [Element: p]
> README.md:21:5 MD033/no-inline-html Inline HTML [Element: br]
> README.md:22:5 MD033/no-inline-html Inline HTML [Element: a]
> README.md:22:79 MD033/no-inline-html Inline HTML [Element: strong]
> README.md:23:5 MD033/no-inline-html Inline HTML [Element: br]
> README.md:24:5 MD033/no-inline-html Inline HTML [Element: br]
> README.md:25:5 MD033/no-inline-html Inline HTML [Element: a]
> README.md:27:5 MD033/no-inline-html Inline HTML [Element: a]
> README.md:29:5 MD033/no-inline-html Inline HTML [Element: a]
> README.md:34:1 MD033/no-inline-html Inline HTML [Element: details]
> README.md:35:3 MD033/no-inline-html Inline HTML [Element: summary]
> README.md:36:3 MD033/no-inline-html Inline HTML [Element: ol]
> README.md:37:5 MD033/no-inline-html Inline HTML [Element: li]
> README.md:38:7 MD033/no-inline-html Inline HTML [Element: a]
> README.md:40:5 MD033/no-inline-html Inline HTML [Element: li]
> README.md:41:7 MD033/no-inline-html Inline HTML [Element: a]
> README.md:43:5 MD033/no-inline-html Inline HTML [Element: li]
> README.md:44:7 MD033/no-inline-html Inline HTML [Element: a]
> README.md:45:7 MD033/no-inline-html Inline HTML [Element: ul]
> README.md:46:9 MD033/no-inline-html Inline HTML [Element: li]
> README.md:46:13 MD033/no-inline-html Inline HTML [Element: a]
> README.md:49:5 MD033/no-inline-html Inline HTML [Element: li]
> README.md:49:9 MD033/no-inline-html Inline HTML [Element: a]
> README.md:57:1 MD033/no-inline-html Inline HTML [Element: p]
> README.md:57:19 MD033/no-inline-html Inline HTML [Element: a]
> README.md:83:1 MD033/no-inline-html Inline HTML [Element: p]
> README.md:83:19 MD033/no-inline-html Inline HTML [Element: a]
> README.md:87:1 MD033/no-inline-html Inline HTML [Element: p]
> README.md:87:19 MD033/no-inline-html Inline HTML [Element: a]
> README.md:97:1 MD033/no-inline-html Inline HTML [Element: p]
> README.md:97:19 MD033/no-inline-html Inline HTML [Element: a]
> README.md:103:1 MD033/no-inline-html Inline HTML [Element: p]
> README.md:103:19 MD033/no-inline-html Inline HTML [Element: a]
> ```

@florianrusch-zf when i did the changes/fixes i also fixed these error manual.

**Comment by florianrusch-zf** [comment]
All linting errors are fixed:

```shell
$ markdownlint-cli2-config .markdownlint.yaml "**/*.md"
markdownlint-cli2-config v0.6.0 (markdownlint v0.27.0)
Finding: **/*.md
Linting: 58 file(s)
Summary: 0 error(s)
```

File: charts/tractusx-connector/values.yaml [hunk 1]
```diff
@@ -273,7 +273,7 @@ controlplane:
   # -- configuration of the [Java Util Logging Facade](https://docs.oracle.com/javase/7/docs/technotes/guides/logging/overview.html)
   logging: |-
     .level=INFO
-    org.eclipse.dataspaceconnector.level=ALL
+    org.eclipse.edc.level=ALL
```
Comment (florianrusch-zf) [inline charts/tractusx-connector/values.yaml:276]: I'm not sure about this. Can someone confirm, that this needs to be changed like this? Maybe @paullatzelsperger or @tuncaytunc-zf

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -43,4 +43,4 @@ Project committers or leaders who do not follow the Code of Conduct in good fait
 
 ## Attribution
 
-This Code of Conduct is adapted from the [Contributor Covenant](https://www.contributor-covenant.org) , version 1.4, available at [https://www.contributor-covenant.org/version/1/4/code-of-conduct.html](https://www.contributor-covenant.org/version/1/4/code-of-conduct/)
\ No newline at end of file
+This Code of Conduct is adapted from the [Contributor Covenant](https://www.contributor-covenant.org) , version 1.4, available at [https://www.contributor-covenant.org/version/1/4/code-of-conduct.html](https://www.contributor-covenant.org/version/1/4/code-of-conduct/)
```
Comment (florianrusch-zf) [inline CODE_OF_CONDUCT.md:46]:
```suggestion
This Code of Conduct is adapted from the [Contributor Covenant](https://www.contributor-covenant.org), version 1.4, available at [https://www.contributor-covenant.org/version/1/4/code-of-conduct.html](https://www.contributor-covenant.org/version/1/4/code-of-conduct/)
```

File: charts/tractusx-connector/values.yaml (hunk 1 above)
Comment (wolf4ood) [reply charts/tractusx-connector/values.yaml:276]: Looks right to me.

**Review by wolf4ood — APPROVED** [review]

File: charts/tractusx-connector/values.yaml (hunk 1 above)
Comment (tuncaytunc-zf) [reply charts/tractusx-connector/values.yaml:276]: "org.eclipse.edc.level=ALL" should be the correct one.

**Review by paullatzelsperger — APPROVED** [review]

**Review by stephanbcbauer — CHANGES_REQUESTED** [review]
Hey @florianrusch-zf as I remember you also added a link to the installation manual in the root readme.md.

Right now I can't find it. Can we add it? Or maybe with a separate PR?

**Comment by florianrusch-zf** [comment]
I haven't add this to the PR as the `INSTALL.md` is not finished yet. As you maybe remember, we discussed this week that the "All-in-one Deployment" was removed and we need to change this.

**Review by stephanbcbauer — APPROVED** [review]
LGTM

**Comment by florianrusch-zf** [comment]
I rebased the branch on the current develop

**Pull request merged**